install_config/persistent_storage/persistent_storage_vsphere.adoc

```diff
@@ -12,23 +12,28 @@
 toc::[]
 
 == Overview
+
 {product-title} supports VMware vSphere's Virtual Machine Disk (VMDK) volumes.
 You can provision your {product-title} cluster with
-xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+xref:../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
```

Comment:
Why are you changing the levels here? I think all of these ../../architecture references were right before.

Reply:
@mylesagray ?  I'm going to roll them back.